instrumentation/rocketmq-client/README.md

```diff
@@ -0,0 +1,98 @@
+# brave-instrumentation-rocketmq-client
```

Comment:
look at the other README examples, I don't recall if we are pasting everything including imports, or just the pertinant parts. I think the latter as it keeps the thing more readable and also we don't compile check the markdown. When reviewing, see if you may have any other ideas to improve the README, but nothing besides this comes to mind.